src/codec_aom.c

```diff
@@ -310,10 +323,21 @@ static avifBool avifProcessAOMOptionsPreInit(avifCodec * codec, struct aom_codec
     return AVIF_TRUE;
 }
 
+typedef enum
+{
+    AVIF_AOM_NUL_OPTION = 0,
+
+    AVIF_AOM_STR_OPTION,
+    AVIF_AOM_INT_OPTION,
+    AVIF_AOM_UINT_OPTION,
+    AVIF_AOM_ENUM_OPTION,
+} avifAOMOptionType;
+
 struct aomOptionDef
 {
     const char * name;
     int controlId;
+    avifAOMOptionType type;
```

Comment:
Please document that if 'type' is AOM_OPTION_ENUM, then 'enums' must be specified.